package/linux-mainline/package

```diff
@@ -21,35 +18,54 @@ sha256sums=(1583d553380a656ecaac8dab89558051e9165983381adc9c3d52fb8ac0ce19fd)
 build() {
     ARCH=arm make imx_v6_v7_defconfig
     ARCH=arm make -j8
+    ARCH=arm make -j8 tools/perf
 }
 
-package() {
-    # Prepare files for the kernel archive
-    local staging="$srcdir"/staging
-    mkdir -p "$staging/boot"
+linux-mainline() {
+    pkgdesc="reMarkable 2 kernel based on the mainline kernel"
+    url=https://www.kernel.org
+    section=kernel
+    installdepends=(kernelctl)
 
-    cp --no-dereference {"$srcdir"/arch/arm,"$staging"}/boot/zImage
-    cp --no-dereference "$srcdir"/arch/arm/boot/dts/imx7d-remarkable2.dtb \
-        "$staging"/boot/zero-sugar.dtb
+    package() {
+        # Prepare files for the kernel archive
+        local staging="$srcdir"/staging
+        mkdir -p "$staging/boot"
 
-    ARCH=arm make -C "$srcdir" modules_install INSTALL_MOD_PATH="$staging"
-    rm "$staging"/lib/modules/*/{source,build}
+        cp --no-dereference {"$srcdir"/arch/arm,"$staging"}/boot/zImage
+        cp --no-dereference "$srcdir"/arch/arm/boot/dts/imx7d-remarkable2.dtb \
+            "$staging"/boot/zero-sugar.dtb
 
-    # Create the kernel archive
-    local archive="mainline-${pkgver%-*}.tar.bz2"
-    install -d "$pkgdir"/opt/usr/share/kernelctl
-    (cd "$staging" && tar --owner root:0 --group root:0 --mtime=$timestamp \
-        -cjf "$pkgdir"/opt/usr/share/kernelctl/"$archive" boot/* lib/modules/*)
+        ARCH=arm make -C "$srcdir" modules_install INSTALL_MOD_PATH="$staging"
+        rm "$staging"/lib/modules/*/{source,build}
+
+        # Create the kernel archive
+        local archive="mainline-${pkgver%-*}.tar.bz2"
+        install -d "$pkgdir"/opt/usr/share/kernelctl
+        (cd "$staging" && tar --owner root:0 --group root:0 --mtime=$timestamp \
+            -cjf "$pkgdir"/opt/usr/share/kernelctl/"$archive" boot/* lib/modules/*)
+    }
+
+    configure() {
+        echo "The new kernel files have been copied, but not installed."
+        echo "Please use kernelctl to select the kernel to boot."
+        echo
+        echo "Known issues with the mainline kernel:"
+        echo " - No support for low power mode (suspend uses more power then it should)"
+        echo " - WiFi sometimes is off on boot (can be turned on again though)"
+        echo " - GUI shutdown in Oxide doesn't work"
+        echo " - Wacom stylus doesn't work in Xochitl (works everywhere else though)"
+        echo " - No OTG control support"
+    }
 }
 
-configure() {
-    echo "The new kernel files have been copied, but not installed."
-    echo "Please use kernelctl to select the kernel to boot."
-    echo
-    echo "Known issues with the mainline kernel:"
-    echo " - No support for low power mode (suspend uses more power then it should)"
-    echo " - WiFi sometimes is off on boot (can be turned on again though)"
-    echo " - GUI shutdown in Oxide doesn't work"
-    echo " - Wacom stylus doesn't work in Xochitl (works everywhere else though)"
-    echo " - No OTG control support"
+linux-mainline-perf() {
```

Comment:
Shouldn't this conflict with `linux-stracciatella-perf` since they provide the same files?